src/refactor_nrepl/ns/libspec_allowlist.clj

```diff
@@ -6,20 +6,29 @@
    (clojure.lang IFn)
    (java.util.regex Pattern)))
 
-(defn- libspec-allowlist* []
-  (let [kondo-file (io/file ".clj-kondo" "config.edn")
-        exclude (when (.exists kondo-file)
-                  (try
-                    (-> kondo-file slurp read-string :linters :unused-namespace :exclude)
-                    (catch Exception e
-                      (when (System/getenv "CI")
-                        (throw e)))))]
-    (->> exclude
-         (mapv (fn [entry]
-                 (if (symbol? entry)
-                   (str "^" (Pattern/quote (str entry)) "$")
-                   entry)))
-         (into (:libspec-whitelist config/*config*)))))
+(defn- kondo-excludes [{:keys [ns meta]}]
+  (let [local-config (:clj-kondo/config meta)
+        local-config (if (and (seq? local-config) (= 'quote (first local-config)))
+                          (second local-config)
+                          local-config)
+        kondo-file (io/file ".clj-kondo" "config.edn")
+        config (when (.exists kondo-file)
+                 (try
+                   (-> kondo-file slurp read-string)
+                   (catch Exception e
+                     (when (System/getenv "CI")
+                       (throw e)))))]
+    (concat (get-in config [:linters :unused-namespace :exclude])
```

Comment:
let's DRY the `[:linters :unused-namespace :exclude]` bit

Reply:
Yeah, wanted to do this from the beginning, but wasn't sure about readability. Done